src/NosCore.Packets/ServerPackets/PinitPacket.cs

```diff
@@ -28,7 +28,7 @@ public class PinitPacket : PacketDefinition
         [PacketIndex(0)]
         public int GroupSize { get; set; }
 
-        [PacketIndex(1, SpecialSeparator = "|")]
+        [PacketIndex(1, IsOptional = true, SpecialSeparator = "|")]
```

Comment:
i don't think this is optional

Reply:
Is optional (I think). When there is only the player without any party the packet sent needs to be "pinit 0", not "pinit 1 1|1|1|etc"

Reply:
right... actually it should be an empty list as 0 is the count